content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -512,6 +514,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
   the input Pod's cpu and memory limits. The intent is to break ties between
   nodes with same scores.
 - `ResourceQuotaScopeSelectors`: Enable resource quota scope selectors.
+- `RootCAConfigMap`: Enable kube-controller-manager to publish a "kube-root-ca.crt" ConfigMap to every namespace. This ConfigMap contains a CA bundle used for verifying connections to the kube-apiserver.
+   See [Bound Service Account Tokens](https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/1205-bound-service-account-tokens/README.md) for more details.
```

Comment:
Can we copy the relevant text from that KEP into pages inside https://kubernetes.io/docs/reference/access-authn-authz/ ?

Reply:
I agree the content should be in the website. I'd actually add it to https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection

Reply:
> add it to https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection

That's a good idea too.
(I'm not keen on making task pages the _only_ place to learn about features, because those can be hard to discover).

Reply:
added to `access-authn-authz/service-accounts-admin.md`